public/js/components/SourceTabs.js

```diff
@@ -12,6 +13,7 @@ const actions = require("../actions");
 const { isEnabled } = require("../../../config/feature");
 
 require("./SourceTabs.css");
+require("./Dropdown.css");
```

Comment:
yeah - that's totally true. I don't want to commit to writing a new Dropdown component though :) I'd rather find a community one